python/lsst/sdm/schemas/efd_lsstcam.yaml

```diff
@@ -0,0 +1,4469 @@
+    "@id": "#exposure_efd.mt_salindex301_temperature_7_stddev"
+    datatype: float
+    nullable: true
+    description: Standard deviation of the weather tower air temperature item 7.
+  - name: "mt_salindex301_temperature_7_min"
+    "@id": "#exposure_efd.mt_salindex301_temperature_7_min"
+    datatype: float
+    nullable: true
+    description: Minimum weather tower air temperature item 7.
+  - name: "mt_salindex301_temperature_7_max"
+    "@id": "#exposure_efd.mt_salindex301_temperature_7_max"
+    datatype: float
+    nullable: true
+    description: Maximum weather tower air temperature item 7.
+  - name: "mt_dimm_fwhm_mean"
+    "@id": "#exposure_efd.mt_dimm_fwhm_mean"
+    datatype: float
+    nullable: true
+    description: Mean combined full width half maximum.
+  - name: "mt_azimuth_encoder_absolute_position_0_mean"
+    "@id": "#exposure_efd.mt_azimuth_encoder_absolute_position_0_mean"
+    datatype: float
+    nullable: true
+    description: Mean Azimuth absolute position read by each encoder head.
+  - name: "mt_azimuth_encoder_absolute_position_1_mean"
+    "@id": "#exposure_efd.mt_azimuth_encoder_absolute_position_1_mean"
+    datatype: float
+    nullable: true
+    description: Mean Azimuth absolute position read by each encoder head.
+  - name: "mt_azimuth_encoder_absolute_position_2_mean"
+    "@id": "#exposure_efd.mt_azimuth_encoder_absolute_position_2_mean"
+    datatype: float
+    nullable: true
+    description: Mean Azimuth absolute position read by each encoder head.
+  - name: "mt_azimuth_encoder_absolute_position_3_mean"
+    "@id": "#exposure_efd.mt_azimuth_encoder_absolute_position_3_mean"
+    datatype: float
+    nullable: true
+    description: Mean Azimuth absolute position read by each encoder head.
+  - name: "mt_elevation_encoder_absolute_position_0_mean"
+    "@id": "#exposure_efd.mt_elevation_encoder_absolute_position_0_mean"
+    datatype: float
+    nullable: true
+    description: Mean Elevation absolute position read by each encoder head.
+  - name: "mt_elevation_encoder_absolute_position_1_mean"
+    "@id": "#exposure_efd.mt_elevation_encoder_absolute_position_1_mean"
+    datatype: float
+    nullable: true
+    description: Mean Elevation absolute position read by each encoder head.
+  - name: "mt_elevation_encoder_absolute_position_2_mean"
+    "@id": "#exposure_efd.mt_elevation_encoder_absolute_position_2_mean"
+    datatype: float
+    nullable: true
+    description: Mean Elevation absolute position read by each encoder head.
+  - name: "mt_elevation_encoder_absolute_position_3_mean"
+    "@id": "#exposure_efd.mt_elevation_encoder_absolute_position_3_mean"
+    datatype: float
+    nullable: true
+    description: Mean Elevation absolute position read by each encoder head.
+  - name: "mt_hexapod_uncompensated_position_u_mean"
+    "@id": "#exposure_efd.mt_hexapod_uncompensated_position_u_mean"
+    datatype: float
+    nullable: true
+    description: Mean U angle.
```

Comment:
"Mean U angle" of what? (Same in subsequent columns.)

Reply:
There are 3 rotational angles in the main telescope hexapod: U, V, and W.